routes/middlewares/festival/lol/get/getLol.ts

```diff
@@ -0,0 +1,35 @@
+import { NextFunction, Request, Response } from 'express';
+
+import CustomError from '@Middleware/error/customError';
+import LolTeam from '@Model/lolTeam.model';
+import LolTeamMember from '@Model/lolTeamMember.model';
+import LolVote from '@Model/lolVote.model';
+
+const getLol = async (req: Request, res: Response, next: NextFunction) => {
```

Comment:
```suggestion
const getLOLTeam.ts = async (req: Request, res: Response, next: NextFunction) => {
```
상황에 따라 카멜케이스를 버려야할 때도 있음, 내가 제안하는 것이 더 readability가 올라가는듯